public/app/app.ts

```diff
@@ -145,6 +147,13 @@ export class GrafanaApp {
       // Provide runRequest implementation to packages, @grafana/scenes in particular
       setRunRequest(runRequest);
 
+      // Privide plugin import utils to packages, @grafana/scenes in particular
+      setPluginImportUtils({
+        importPanelPlugin,
+        importPanelPluginFromMeta,
+        syncGetPanelPlugin,
```

Comment:
```suggestion
        tryGetPanelPluginFromCache,
```

Not 100% we need this, the way we use it from VizPanel right now does not actually save a render (it does save a render as it's used from PanelRenderer). If we changed so that we used it from VizPanel constructor (and also then loaded defaults etc from there it would save a render cycle, but not sure that is worth it)